### PR TITLE
06 view article

### PR DESCRIPTION
**Pull request by ZanClifton**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -70,6 +70,35 @@ body {
...
+
+.card-title a:hover {
+  text-decoration: none;
+  color: #077007;
+  font-size: 1.005em;
+}
+
+.an-article {
+  gap: 30px;
+  min-width: 500px;
```
Comment (corkesw) [inline src/App.css:102]: Check out what this looks like on a mobile screen.

File: src/components/AnArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import { useParams } from "react-router-dom";
+import { getAnArticle } from "../utils/api";
+
+export const AnArticle = () => {
+  const [article, setArticle] = useState({});
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getAnArticle(article_id).then((articleFromApi) => {
+      setArticle(articleFromApi);
```
Comment (corkesw) [inline src/components/AnArticle.jsx:11]: add a catch block to deal with a non-existent article_id

**Comment by corkesw** [comment]
Have you missed out ticket 5? Can't see any functionality around topics yet. The single article component looks good, just needs some error handling to catch any 404s. Have a look at the styling - use dev tools to see this in a mobile view as the content goes off the edge of the screen. You may plan to add media queries later which is fine.

File: src/components/AnArticle.jsx (hunk 2 above)
Comment (ZanClifton) [reply src/components/AnArticle.jsx:11]: Added

File: src/App.css (hunk 1 above)
Comment (ZanClifton) [reply src/App.css:102]: Unable to resolve. Will reformat design at a later stage.

**Comment by ZanClifton** [comment]
No tickets missed. Branch created before ticket 5 PR completed.

**Pull request merged**